grpc/tests/src/test/regression/ee_1119.rs

```diff
@@ -0,0 +1,270 @@
+use casper_engine_test_support::{
+    internal::{
+        utils, ExecuteRequestBuilder, InMemoryWasmTestBuilder, DEFAULT_ACCOUNTS,
+        DEFAULT_ACCOUNT_PUBLIC_KEY, DEFAULT_LOCKED_FUNDS_PERIOD,
+    },
+    DEFAULT_ACCOUNT_ADDR, DEFAULT_ACCOUNT_INITIAL_BALANCE, MINIMUM_ACCOUNT_CREATION_BALANCE,
+};
+use casper_execution_engine::{core::engine_state::genesis::GenesisAccount, shared::motes::Motes};
+use casper_types::{
+    account::AccountHash,
+    auction::{
+        Bids, UnbondingPurses, ARG_DELEGATOR, ARG_UNBOND_PURSE, ARG_VALIDATOR,
+        ARG_VALIDATOR_PUBLIC_KEYS, BIDS_KEY, METHOD_RUN_AUCTION, METHOD_SLASH,
+        UNBONDING_PURSES_KEY,
+    },
+    mint::TOTAL_SUPPLY_KEY,
+    runtime_args, PublicKey, RuntimeArgs, URef, U512,
+};
+use once_cell::sync::Lazy;
+
+const CONTRACT_TRANSFER_TO_ACCOUNT: &str = "transfer_to_account_u512.wasm";
+const CONTRACT_WITHDRAW_BID: &str = "withdraw_bid.wasm";
+const CONTRACT_DELEGATE: &str = "delegate.wasm";
+const CONTRACT_UNDELEGATE: &str = "undelegate.wasm";
+
+const DELEGATE_AMOUNT_1: u64 = 95_000;
+const UNDELEGATE_AMOUNT_1: u64 = 17_000;
+
+const TRANSFER_AMOUNT: u64 = MINIMUM_ACCOUNT_CREATION_BALANCE;
+
+const ARG_AMOUNT: &str = "amount";
+const ARG_PUBLIC_KEY: &str = "public_key";
+
+const SYSTEM_ADDR: AccountHash = AccountHash::new([0u8; 32]);
+const VALIDATOR_1: PublicKey = PublicKey::Ed25519([3; 32]);
+static VALIDATOR_1_ADDR: Lazy<AccountHash> = Lazy::new(|| VALIDATOR_1.into());
+const VALIDATOR_1_STAKE: u64 = 250_000;
+
+#[ignore]
+#[test]
+fn should_run_ee_1119_dont_slash_delegated_validators() {
+    let accounts = {
+        let validator_1 = GenesisAccount::new(
+            VALIDATOR_1,
+            *VALIDATOR_1_ADDR,
+            Motes::new(DEFAULT_ACCOUNT_INITIAL_BALANCE.into()),
+            Motes::new(VALIDATOR_1_STAKE.into()),
+        );
+
+        let mut tmp: Vec<GenesisAccount> = DEFAULT_ACCOUNTS.clone();
+        tmp.push(validator_1);
+        tmp
+    };
+    let run_genesis_request = utils::create_run_genesis_request(accounts);
+
+    let mut builder = InMemoryWasmTestBuilder::default();
+    builder.run_genesis(&run_genesis_request);
+
+    let exec_request = ExecuteRequestBuilder::standard(
```

Comment:
Could we give these more descriptive names, like "fund_system_exec_request`?

(and below with `..._2`, `..._3`, etc.)